app/src/main/java/com/jerboa/ui/components/settings/lookandfeel/LookAndFeelActivity.kt

```diff
@@ -330,6 +334,23 @@ fun LookAndFeelActivity(
                         )
                     },
                 )
+                SettingsList(
+                    title = {
```

Comment:
I know its somewhat arbitrary, but I think the settings page looks *much* better when the types are grouped. Could you move the `Confirm exit` and `Post Actionbar Mode`, belowe the `Post View` setting, so that all the list-type settings are grouped above the boolean settings?

Reply:
I moved them